src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -135,7 +136,11 @@ public SafeProcessHandle SafeHandle
         /// <internalonly/>
         private bool Associated
         {
-            get { return _haveProcessId || _haveProcessHandle; }
+            get
+            {
+                CheckDisposed();
```

Comment:
This property didn't throw before. It's also private, why would we need to guard calls to it?

Reply:
removed from private